adev/src/app/features/not-found/not-found.html

```diff
@@ -0,0 +1,80 @@
+<main class="docs-viewer">
+  <header class="docs-header">
+    <div class="docs-page-title">
+      <h1 tabindex="-1"><span class="docs-emoji">Page Not Found 🙃</span></h1>
+    </div>
+  </header>
+  <p>
+    We couldn't find what you were looking for. We have initiated a search for the term extracted
+    from the URL.
+  </p>
+  <p>
+    If you think this is a mistake, please
+    <a
+      href="https://github.com/angular/angular/issues/new?template=3-docs-bug.yaml"
+      target="_blank"
+    >
+      open an issue</a
+    >
+    so we can fix it.
+  </p>
+
+  <!-- Search results that match the queried URL that we couldn't find  -->
+  @if (searchResults().length > 0) {
+    <!-- The search results are faded in if/once we have results -->
+    <div animate.enter="enter-animation">
+      <br /><br />
+      <h2>Feeling lucky?</h2>
+      <p>Maybe what you're looking for is in the list below:</p>
+
+      <ul class="docs-search-results docs-mini-scroll-track">
+        @for (result of searchResults(); track $index) {
+          <li docsSearchItem [item]="result" class="docs-search-result">
+            <a
+              [routerLink]="'/' + result.url | relativeLink: 'pathname'"
+              [fragment]="result.url | relativeLink: 'hash'"
+            >
```

Comment:
We need to force the router link to be relative to the root.

```suggestion
           <a 
              [relativeTo]="null"
              [routerLink]="'/' + result.url | relativeLink: 'pathname'"
              [fragment]="result.url | relativeLink: 'hash'"
            >
```